src/controllers/controller.polarArea.js

```diff
@@ -211,12 +225,30 @@ module.exports = function(Chart) {
 			return count;
 		},
 
-		calculateCircumference: function(value) {
+		_computeAngle: function(index) {
+			var me = this;
 			var count = this.getMeta().count;
-			if (count > 0 && !isNaN(value)) {
-				return (2 * Math.PI) / count;
+			var dataset = me.getDataset();
+			var meta = me.getMeta();
+			var value = dataset.data[index];
+			var isHidden = meta.data[index].hidden;
+
+			if (count <= 0 || isNaN(value) || isHidden) {
```

Comment:
When referenced only one time, I think it's preferable to avoid local variable for short assignment. Also, I don't think we need to check `count <= 0` anymore because (if I'm not wrong) if `count <= 0` then the value is either `NaN` or `hidden`.

```javascript
if (isNaN(dataset.data[index]) || meta.data[index].hidden) {
  return 0;
}
```